src/Microsoft.DocAsCode.Glob/FileGlob.cs

```diff
@@ -34,9 +34,15 @@ private static IEnumerable<string> GetFilesCore(string cwd, IEnumerable<GlobMatc
 
         private static IEnumerable<string> GetFilesFromSubfolder(string baseDirectory, string cwd, IEnumerable<GlobMatcher> globs, IEnumerable<GlobMatcher> excludeGlobs)
         {
+            var relativePath = GetRelativeFilePath(cwd, baseDirectory);
```

Comment:
When I wanna exclude node_modules folder? should it be `node_modules/` instead of `node_modules/**`? Consider the following case: `exclude`: `**/node1`, while there are folder `node1` and file `a/node2`, folder `node1` should not be excluded.